dashboards/google-cloud-agent-for-sap/metadata.yaml

```
@@ -1,10 +1,26 @@
 sample_dashboards:
+  -
+    category: Agent for SAP
+    id: agent-for-sap-hana-availability-monitoring
+    display_name: Agent for SAP - HANA Availability Monitoring
+    description: "This dashboard is based on [Google Cloud’s Agent for SAP](https://cloud.google.com/solutions/sap/docs/agent-for-sap/latest/planning). It provides an overview of the availability of an SAP HANA system. The availability of a standalone SAP HANA system is derived from the status of the instance's services, such as `hdbindexserver` and `hdbnameserver`. For highly-available SAP HANA deployments, the availability is also derived from the replication status between the primary and secondary sites, and also shows the status of the individual Pacemaker resource agents."
```

```suggestion
    description: "This dashboard is based on [Google Cloud’s Agent for SAP](https://cloud.google.com/solutions/sap/docs/agent-for-sap/latest/all-guides). It provides an overview of the availability of an SAP HANA system. The availability of a standalone SAP HANA system is derived from the status of the instance's services, such as `hdbindexserver` and `hdbnameserver`. For highly-available SAP HANA deployments, the availability is also derived from the replication status between the primary and secondary sites, and also shows the status of the individual Pacemaker resource agents."
```

Should this link be the same as the links for the other dashboards for this integration?